{"name": "test_LongParam", "status": "broken", "statusDetails": {"message": "ConnectionResetError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。", "trace": "self = <class 'test_plugins.Test_plugins'>\n\n    def setup_class(self):\n        self.p1 = plugins(\"192.168.1.1\",\"23\",\"telnetadmin\",\"telnetadmin\")\n>       self.p1.initialize()\n\ntest_plugins.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nlogin.py:132: in initialize\n    self.tel_debug_FM104()\nlogin.py:59: in tel_debug_FM104\n    self.tn.read_until(b\"login: \",timeout=1)\nD:\\python 3.8.7\\lib\\telnetlib.py:317: in read_until\n    self.fill_rawq()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <telnetlib.Telnet object at 0x000001C8F52C61C0>\n\n    def fill_rawq(self):\n        \"\"\"Fill raw queue from exactly one recv() system call.\n    \n        Block if no data is immediately available.  Set self.eof when\n        connection is closed.\n    \n        \"\"\"\n        if self.irawq >= len(self.rawq):\n            self.rawq = b''\n            self.irawq = 0\n        # The buffer size should be fairly small so as to avoid quadratic\n        # behavior in process_rawq() above\n>       buf = self.sock.recv(50)\nE       ConnectionResetError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。\n\nD:\\python 3.8.7\\lib\\telnetlib.py:526: ConnectionResetError"}, "start": 1700639028726, "stop": 1700639028726, "uuid": "2321861a-9979-470c-9ae1-dcd0f91fcc30", "historyId": "ada529718f23cb5a4760f3e163b5afbe", "testCaseId": "ada529718f23cb5a4760f3e163b5afbe", "fullName": "test_plugins.Test_plugins#test_LongParam", "labels": [{"name": "suite", "value": "test_plugins"}, {"name": "subSuite", "value": "Test_plugins"}, {"name": "host", "value": "l29707B"}, {"name": "thread", "value": "8116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_plugins"}]}